src/NotSureWhere/parseLaMEM.jl

```diff
@@ -0,0 +1,27 @@
+using LaMEM, GeophysicalModelGenerator
+
+# structure definition
+struct markerGrid
+    Xpart::Array{Float64,3}
+    Ypart::Array{Float64,3}
+    Zpart::Array{Float64,3}
+    Temp::Array{Float64,3}
+    Phase::Array{Int64,3}
+end
+
+markerGrid(X, Y, Z) = markerGrid(X, Y, Z, zeros(size(X)), zeros(Int64, size(X)))
+
+
+# function definition
+function createMarkerGrid(filename::String, numCores::Int64)
```

Comment:
In general there is no need to add type annotations in the function call. Exceptions are when you really want to be restrictive or you want to use multiple dispatch.

Reply:
Good to know. I guess in this case, being restrictive is good because I cannot see any other combination that would be reasonable